### PR TITLE
updated the code to django 2.1/python 3.6 and fixed some issues

### PR DESCRIPTION
**Pull request by prashantsinalkar**

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -2,23 +2,14 @@
 import os
 import sys
 
-if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "pyfoss.settings")
-
+if __name__ == '__main__':
```
Comment (prathamesh920) [inline manage.py:5]: ```manage.py``` file is generated by django.
This should not be modified.
You should create a new project using django 2.1 and clone the source code only.

File: pyfoss/settings.py [hunk 2]
```diff
@@ -1,210 +1,159 @@
...
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+PROJECT_DIR = os.path.abspath(os.path.dirname(__file__) + '/../')
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '!-c13kb5-x5tbp)tnllico+a7++a2r(_b-20xfc^#5x$tlmtv1'
```
Comment (prathamesh920) [inline pyfoss/settings.py]: keep this secret

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-configparser==3.5.0
-Django==1.10
+Django==2.1.7
 django-nested-inline==0.3.7
-mysqlclient==1.3.10
+mysqlclient==1.4.2.post1
+pytz==2018.9
```
Comment (prathamesh920) [inline requirements.txt]: I think ```pytz``` will come along with django itself.

File: static/website/templates/content.html [hunk 4]
```diff
@@ -1,3 +1,3 @@
 {% block content %}
-<h1>Contenet</h1>
-{% endblock%}
+<h1>Contene</h1>
```
Comment (prathamesh920) [inline static/website/templates/content.html]: typo

File: website/models.py [hunk 5]
```diff
@@ -1,72 +1,92 @@
...
+
 
 class Nav(models.Model):
-	"""Main navigation links"""
-	nav_name = models.CharField(max_length=200)
-	link = models.CharField(max_length=200)
-	position = models.IntegerField()
-	visible = models.BooleanField()
+    """Main navigation links"""
+    nav_name = models.CharField(max_length=200)
```
Comment (prathamesh920) [inline website/models.py:6]: Is it possible for u to change the name ```nav_name``` to simply ```name```

File: website/models.py [hunk 6]
```diff
@@ -1,72 +1,92 @@
 from django.db import models
+from datetime import date
```
Comment (prathamesh920) [inline website/models.py]: Use django's timezone package instead

File: website/models.py [hunk 7]
```diff
@@ -1,72 +1,92 @@
...
 class SubNav(models.Model):
-	"""Navigation links for pages"""
-	nav = models.ForeignKey(Nav)
-	subnav_name = models.CharField(max_length=200)
-	link = models.CharField(max_length=200)
-	position = models.IntegerField()
-	visible = models.BooleanField()
+    """Navigation links for pages"""
+    nav = models.ForeignKey(Nav, on_delete=models.CASCADE)
+    subnav_name = models.CharField(max_length=200)
```
Comment (prathamesh920) [inline website/models.py:18]: same here, simply name

**Review by prathamesh920 — CHANGES_REQUESTED** [review]
Kindly resolve the comments.
Also I saw PHP files! So this website depends on PHP  I mean has PHP requirements as well!
So try a fresh deployment and check whether all works well, and what if machine don't have PHP required.
Also, why not only python-django for django website, why PHP dependency?

File: website/models.py (hunk 5 above)
Comment (prashantsinalkar) [reply website/models.py:6]: No.

File: website/models.py (hunk 7 above)
Comment (prashantsinalkar) [reply website/models.py:18]: No

File: manage.py (hunk 1 above)
Comment (prashantsinalkar) [reply manage.py:5]: Git showing the difference only. It is the default file created by django 2.1.7

**Comment by prashantsinalkar** [comment]
> Kindly resolve the comments.
> Also I saw PHP files! So this website depends on PHP I mean has PHP requirements as well!
> So try a fresh deployment and check whether all works well, and what if machine don't have PHP required.
> Also, why not only python-django for django website, why PHP dependency?
For Your information, it is the mandatory for to collect stats.

File: website/models.py (hunk 6 above)
Comment (prashantsinalkar) [reply website/models.py]: fixed.

File: static/website/templates/content.html (hunk 4 above)
Comment (prashantsinalkar) [reply static/website/templates/content.html]: fixed.

File: requirements.txt (hunk 3 above)
Comment (prashantsinalkar) [reply requirements.txt]: fixed.

**Comment by prathamesh920** [comment]
> > Kindly resolve the comments.
> > Also I saw PHP files! So this website depends on PHP I mean has PHP requirements as well!
> > So try a fresh deployment and check whether all works well, and what if machine don't have PHP required.
> > Also, why not only python-django for django website, why PHP dependency?
> > For Your information, it is the mandatory for to collect stats.

That is fine but why in PHP?
So PHP is also requirements for this project, right?

**Comment by prashantsinalkar** [comment]
> > > Kindly resolve the comments.
> > > Also I saw PHP files! So this website depends on PHP I mean has PHP requirements as well!
> > > So try a fresh deployment and check whether all works well, and what if machine don't have PHP required.
> > > Also, why not only python-django for django website, why PHP dependency?
> > > For Your information, it is the mandatory for to collect stats.
> 
> That is fine but why in PHP?
> So PHP is also requirements for this project, right?

No. Without PHP also it works fine. FYI PHP and Django are independent, and SYS-ads will take care. I think you are not aware with the pyfoss structure.

**Comment by prathamesh920** [comment]
In that case my point is PHP files should not be in this repo!

**Comment by prathamesh920** [comment]
Merging for now.

**Review by prathamesh920 — APPROVED** [review]

**Pull request merged**